### ISSUE TITLE
Alt text is not descriptive [/src/**/start files] 

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7063]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/angular/hosting.mdx:
```
  43  
  44: ![image](/images/browser-published-updated.png)
  45  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/flutter/integrate.mdx:
```
  212  
  213: ![Budget Tracker home screen](/images/lib/getting-started/flutter/budget-tracker-home.png)
  214  

  290  
  291: ![Budget Tracker with Authenticator](/images/lib/getting-started/flutter/budget-tracker-authenticator.png)
  292  

  493  
  494: ![Budget Tracker create screen](/images/lib/getting-started/flutter/budget-tracker-create-screen.png)
  495  

  573  
  574: ![Completed budget tracker](/images/lib/getting-started/flutter/budget-tracker-complete.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/ios/setup.mdx:
```
  53  
  54:    ![Add package dependency](/images/lib/getting-started/ios/set-up-ios-add-package-dependency.png)
  55  

  59  
  60:    ![Add the Amplify package](/images/lib/getting-started/ios/set-up-ios-search-amplify-repo.png)
  61  

  63  
  64:    ![Select dependencies](/images/lib/getting-started/ios/set-up-ios-select-dependencies.png)
  65  

  73  
  74: ![Edit dependencies](/images/lib/getting-started/ios/set-up-ios-edit_dependencies.png)
  75  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/next/hosting.mdx:
```
  24  
  25: ![image](/images/start-nextjs-deploy-1.png)
  26  

  28  
  29: ![image](/images/start-nextjs-deploy-2.png)
  30  

  34  
  35: ![image](/images/start-nextjs-deploy-3.png)
  36  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/reactnative/generate-model.mdx:
```
  12  
  13:    ![Create new schema](/images/lib/getting-started/reactnative/generate-model-admin-ui-create.png)
  14  

  23  
  24:    ![Add isComplete field](/images/lib/getting-started/reactnative/generate-model-admin-ui-isComplete-field.png)
  25  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/start/getting-started/vanillajs/hosting.mdx:
```
  29  
  30: ![image](/images/browser-vanillajs-hosting.png)
  31  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/start/getting-started/hosting/index.mdx:
```
  101  
  102: ![image](/images/browser-vanillajs-hosting.png)
  103  

  107  
  108: ![image](/images/browser-published-updated.png)
  109  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/start/getting-started/hosting/index.mdx:
```
  101  
  102: ![image](/images/browser-vanillajs-hosting.png)
  103  

  107  
  108: ![image](/images/browser-published-updated.png)
  109  
```
